blog/2023-03-17-project-coordinator.md

```diff
@@ -0,0 +1,38 @@
+---
+author: Antonin Delpeuch
+title: "OpenRefine is looking for a part time project coordinator"
+---
+
+OpenRefine is looking for a part time project coordinator.
+This paid role is intended to help facilitate community processes in the project, improve our capacity to liaise with partners and our governance structures.
```

Comment:
Who are "partners" in this context? Since this references governance structures, would it make sense to get those finalized before posting this?

Reply:
By partners I mean CS&S itself, funders, internship programmes we participate in, organizations around us (non-profits like the Wikimedia Foundation or chapters, companies like OntoText or OpenCorporates)…
About governance, see below.